openedx/core/djangoapps/organization_api/api/views.py

```diff
@@ -0,0 +1,45 @@
+"""
+Organizations API views.
+"""
+from rest_framework import permissions
+from rest_framework.authentication import SessionAuthentication
+from rest_framework.response import Response
+from rest_framework.status import HTTP_404_NOT_FOUND
+from rest_framework.views import APIView
+from rest_framework_oauth.authentication import OAuth2Authentication
+
+from util.organizations_helpers import get_organization_by_short_name
+
+
+class OrganizationsView(APIView):
+    """
+    View to get organization information.
+    """
+    authentication_classes = (OAuth2Authentication, SessionAuthentication,)
+    permission_classes = (permissions.IsAuthenticated,)
+
+    def get(self, request, organization_key):
+        """
+        Return organization information related to provided organization
+        key/short_name.
+        """
+        organization = get_organization_by_short_name(organization_key)
+        if organization:
+            organization_data = {
+                'organization_name': organization.get('name', ''),
```

Comment:
These fields clearly relate to an organization. As such, the field names don't all need to be prefixed with "organization_".